pravega/pravega
# Issue 6845: Remove build warnings

**Change log description**  
- Removes warnings when building the project
- Improves ease of use of documentation code snippets

**Purpose of the change**  
- Closes #6845
- Closes #6823 

**What the code does**  
- Added param and return tags to variables that gave warnings after delombok process in javadoc task in build script.
- Fixed typo in SynchronizerConfig
- Removed some `$` signs in documentation code snippets for easier copying/pasting to command line

**How to verify it**  
- Build passes without warnings
- See #6823
